### PR TITLE
fix/#160-run-all-tests-in-ci

### PR DESCRIPTION
**Pull request by smnws**

### CODE REVIEW

**Comment by smnws** [comment]
CI / k8s-cypress-tests fails sometime. [example](https://github.com/amosproj/amos2023ss04-kubernetes-inventory-taker/actions/runs/5371540079/jobs/9744368383)

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -81,14 +91,48 @@ jobs:
...
+          if-no-files-found: ignore
+          retention-days: 1
+
+      - name: Cypress run component testing
+        uses: cypress-io/github-action@v5
+        with:
+          working-directory: Explorer
+          component: true
+
+
```
Comment (vringar) [inline .github/workflows/ci.yaml:136]: So when component tests fail we don't need screenshots?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -81,14 +91,48 @@ jobs:
   #unit-test:
   #runs-on: "ubuntu-latest"
 
-  setup-kubernetes-cluster:
+  k8s-cypress-tests:
```
Comment (vringar) [inline .github/workflows/ci.yaml:94]: The way this is currently set up makes this an end-to-end test, so maybe we should make that explicit in the job name
```suggestion
  e2e-testing:
```

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (vringar) [reply .github/workflows/ci.yaml]: I just saw that the failing tests also produce a video, should we save that as well?

Comment (smnws) [reply .github/workflows/ci.yaml]: already done

**Pull request merged**